navigators/LoggedInNav.js

```diff
@@ -14,32 +11,35 @@ export default function LoggedInNav() {
       screenOptions={{
         tabBarActiveTintColor: "white",
         tabBarShowLabel: false,
+        headerShown: false,
         tabBarStyle: {
           borderTopColor: "rgba(255, 255, 255, 0.3)",
           backgroundColor: "black",
         },
       }}
     >
       <Tabs.Screen
-        name="Feed"
-        component={Feed}
+        name="TabFeed"
```

Comment:
<img width="551" alt="스크린샷 2022-05-29 오후 4 08 42" src="https://user-images.githubusercontent.com/60570733/170857224-235cbb25-b6d8-481a-bac4-55c6097dfb20.png">
`This can cause confusing behavior during navigation. Consider using unique names for each screen instead.`

위와 같은 경고 뜨는 경우는 `Tab Screen`의 name과 `Stack Screen`의 name이 같아서 생기는 경고.
=> 두 스크린의 이름을 다르게 변경.